Core/include/Acts/Propagator/Propagator.ipp

```diff
@@ -26,23 +25,22 @@ auto Acts::Propagator<S, N>::propagate_impl(propagator_state_t& state,
   // Navigator initialize state call
   m_navigator.initialize(state, m_stepper);
   // Pre-Stepping call to the action list
-  state.options.actionList(state, m_stepper, m_navigator, result, logger());
```

Comment:
I like this - removes this awkward circular stuffing back of the result to the one that would create the result.